timemachine/fe/mcgregor.py

```diff
@@ -411,6 +421,35 @@ def atom_map_pop(map_1_to_2, map_2_to_1, idx, jdx):
     map_2_to_1[jdx] = UNMAPPED
 
 
+def is_chiral_conversion(bond_graph: nx.Graph, disabled_bonds: Set[Tuple[int, int]]) -> bool:
+    for node in bond_graph.nodes():
+        nbs = list(nx.neighbors(bond_graph, node))
+        if len(nbs) == 4:
```

Comment:
Thinking about this some more based on @maxentile 's comment that we also treat certain 3-connected tertiary nitro groups as non-invertible (where disabled_bonds_count > 0 would trigger the assertion). Should at least add a `TODO` here at the very minimum.

Reply:
> Thinking about this some more based on @maxentile 's comment that we also treat certain 3-connected tertiary nitro groups as chiral centers

[clarification] My comment Monday was that your notes covered the `X4` case but not the `X3` case. I don't think the current list of smarts patterns in `find_chiral_atoms` https://github.com/proteneer/timemachine/blob/3a174e63d32f4813f2d6cf4e9de4a677e6ba7419/timemachine/fe/chiral_utils.py#L146-L148 matches nitro groups.

[agreement] +1 on adding an inline `# TODO` for the `X3` case, if not covered in this PR. (I think @mcwitt listed this as a todo in the PR description yesterday. I do not have input on how to extend this logic to `len(nbs) == 3`.)

Reply:
Ah thanks for the clarification - I misremembered.

Reply:
Added TODO comment in https://github.com/proteneer/timemachine/pull/1356/commits/9ec19c8f94a1feb838a4ae3765edd33eb6628382

Reply:
Since https://github.com/proteneer/timemachine/pull/1356/commits/546037d42cb12ff9af3303002f77d09c122c2ef9, we use the same code path for verifying chiral consistency in the MCS leaf filter and in the `SingleTopology` assertions (except for the former using a placeholder forcefield for speed, while the latter uses OFF 2.0). So I think this is no longer an issue.